roles/reproducer/tasks/devscripts_layout.yml

```diff
@@ -1,6 +1,223 @@
+      }}
+  block:
+    - name: Create blank images for OCP cluster
+      ansible.builtin.import_role:
+        name: "libvirt_manager"
+        tasks_from: "ocp_layout.yml"
+
+    - name: Refresh VBMC data
+      ansible.builtin.import_role:
+        name: "virtualbmc"
+        tasks_from: "list_hosts.yml"
+
+    - name: Generate baremetal content fact
+      vars:
+        _hypervisor_ip4: >-
+          {{ hostvars[inventory_hostname].ansible_default_ipv4.address }}
+        _vm_data: "{{ _cifmw_libvirt_manager_layout.vms.ocp }}"
+        _mac: >-
+          {{
+            cifmw_libvirt_manager_mac_map[item] |
+            selectattr('network', 'equalto', cifmw_reproducer_provision_net) |
+            map(attribute='mac') | first
+          }}
+        _matcher: >-
+          {{ cifmw_devscripts_config.cluster_name }}_master_{{ vm_id }}
+        _ipmi_port: >-
+          {{
+            cifmw_virtualbmc_known_hosts |
+            selectattr('Domain name', 'match', _matcher) |
+            map(attribute='Port') | first
+          }}
+        _node:
+          name: "openshift-master-{{ vm_id  }}"
+          driver: "ipmi"
+          resource_class: "baremetal"
+          driver_info:
+            username: "admin"
+            password: "password"
+            address: "ipmi://{{ _hypervisor_ip4 }}:{{ _ipmi_port }}"
+            deploy_kernel: "http://172.22.0.2/images/ironic-python-agent.kernel"
+            deploy_ramdisk: >-
+              http://172.22.0.2/images/ironic-python-agent.initramfs
+            disable_certificate_verification: false
+          ports:
+            - address: "{{ _mac }}"
+              pxe_enabled: true
+          properties:
+            local_gb: "{{ (_vm_data.disksize | default(105) | int) - 5}}"
+            cpu_arch: "x86_64"
+            boot_mode: "UEFI"
+      ansible.builtin.set_fact:
+        _ironic_bm_data: >-
+          {{
+            (_ironic_bm_data | default([])) + [_node]
+          }}
+      loop: >-
+        {{ cifmw_libvirt_manager_all_vms | select('match', 'ocp-[0-9]+') }}
+      loop_control:
+        index_var: vm_id
+        label: "{{ item }}"
+
+    - name: Output baremetal content in file
+      vars:
+        _content:
+          nodes: "{{ _ironic_bm_data }}"
+      ansible.builtin.copy:
+        content: "{{ _content | to_nice_json }}"
+        dest: "{{ _bm_ironic_file }}"
+        mode: "0644"
+
+    - name: Group devscripts last actions before deployment
+      vars:
+        _pub_net_data: >-
+          {{
+            _libvirt_manager_networking.networks[cifmw_libvirt_manager_pub_net]
+          }}
+      block:
+        - name: Generate patched overrides for devscripts
+          vars:
+            _bootstrap_mac: "52:54:ab:83:31:87"
+          ansible.builtin.set_fact:
+            cifmw_devscripts_config_overrides_patch_01_baremetal:
+              external_bootstrap_mac: "{{ _bootstrap_mac }}"
+              manage_br_bridge: 'n'
```

Comment:
Please add `manage_int_bridge: 'n'`... there was a check for this leading to conflicts.